src/components/EntityInfo.vue

```diff
@@ -289,50 +311,62 @@ export default class EntityInfo extends Mixins(CommonMixin, EnumMixin) {
   }
 }
 
-// ENABLE THIS TO GET STAFF-SPECIFIC BACKGROUND IMAGE
-// #entity-info.staff {
-//   background-image: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='105' height='100'><text x='0' y='105' font-size='30' transform='rotate(-45 10,40)' opacity='0.1'>STAFF</text></svg>");
-//   background-repeat: repeat-x;
-// }
-
 #entity-legal-name,
 #incorp-app-title {
   display: inline-block;
   color: $gray9;
   letter-spacing: -0.01rem;
-  font-size: 1.125rem;
+  font-size: 1.375rem;
   font-weight: 700;
   text-transform: uppercase;
 }
 
 #entity-status {
-  margin-top: 0.25rem;
+  margin-top: 5px;
   margin-left: 0.5rem;
   vertical-align: top;
 }
 
+#entity-description,
+#nr-subtitle {
+  font-size: 0.875rem;
+  color: $gray9;
+}
+
+// vertical lines between buttons:
+menu > span + span {
+  border-left: 1px solid $gray3;
+  border-radius: 0;
+}
+
 dl {
-  display: inline-block;
-  overflow: hidden;
-  color: $gray6;
+  font-size: 0.875rem;
+  line-height: 1.5rem;
 }
 
-dd, dt {
+dt {
+  color: $gray9;
+  font-weight: bold;
   float: left;
+  clear: left;
+  margin-right: 0.5rem;
 }
 
-dt {
-  position: relative;
+dd {
+  // margin-left: 0;
+  cursor: pointer;
 }
 
-dd + dt:before {
-  content: "•";
-  display: inline-block;
-  margin-right: 0.75rem;
-  margin-left: 0.75rem;
+// hide change buttons when not hovering on value:
+dd:not(:hover) > button {
+  display: none;
```

Comment:
Special UX request: don't show the Change buttons until the user hovers over that area.